_automation/main.go

```diff
@@ -46,7 +47,7 @@ func (g *Grammar) ContentURL() string {
 }
 
 func (g *Grammar) FetchNewVersion() *GrammarVersion {
-	if strings.HasPrefix(g.Reference, "v") {
+	if strings.HasPrefix(g.Reference, "v") || strings.EqualFold(g.UpdatedBasedOn, "tag") {
```

Comment:
let's remove `strings.HasPrefix(g.Reference, "v")` and use `g.UpdatedBasedOn` to choose between tag&commit instead